charts/argo-rollouts/templates/controller/deployment.yaml

```diff
@@ -44,6 +44,7 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       serviceAccountName: {{ include "argo-rollouts.serviceAccountName" . }}
+      automountServiceAccountToken: {{ .Values.serviceAccount.automount }}
```

Comment:
Since we are setting the field on the `kind: Deployment` rather than on the `kind: ServiceAccount` can we use `{{ .Values.controller.automountServiceAccountToken }}` instead?